src/node_file.cc

```diff
@@ -2826,6 +3109,19 @@ InternalFieldInfoBase* BindingData::Serialize(int index) {
   return info;
 }
 
+void BindingData::CreatePerIsolateProperties(IsolateData* isolate_data,
+                                             Local<ObjectTemplate> target) {
+  Isolate* isolate = isolate_data->isolate();
+
+  SetMethod(
```

Comment:
```suggestion
  SetMethodNoSideEffect(
```

Reply:
Calling `uv` is not considered as side effect?

Reply:
You're right.